nss-day-cohort-30/bangazon-site-cunning-carps
# Photo upload

#Description
Users can upload photos for their products and view the photo in the details page.
#Type of change
New feature (non-breaking change which adds functionality)

 
#Testing Instructions for Change Made
git fetch --all
git checkout PhotoUpload
dotnet run
verify...

#Checklist:
 My code follows the style guidelines of this project - yes
 I have performed a self-review of my own code - yes
 I have commented my code, particularly in hard-to-understand areas - yes
 My changes generate no new warnings - yes
 I have added test instructions that prove my fix is effective or that my feature works - yes
